paper/PAPER.typst

```diff
@@ -41,20 +41,22 @@ HVM2's syntax has seven different types of _agents_ (in Lafont's terms) or _Node
   ```
   <Node> ::=
       | "*"                    -- (ERA)ser
-      | "@" <name>             -- (REF)erence
-      | <number>               -- (NUM)ber
+      | "@" <alphanumeric>     -- (REF)erence
```

Comment:
If you want to be more specific about "alphanumeric", the regex we're using is `[a-zA-Z0-9_.-/]+`

Reply:
Can defs start with digits? `@0 = ..`?

Reply:
yup